docs/040-employee-handbook-us/benefits-and-holidays.md

```diff
@@ -10,7 +10,8 @@ CivicActions is considered closed on the following holidays:
 | ----------------------------------- | ----------------------------- |
 | New Year's Day                      | January 1st\*                 |
 | Birthday of Martin Luther King, Jr. | Third Monday in January       |
-| Memorial Day                        | Last Monday in May            |
+| Memorial Day                        | Last Monday in May 
+| Juneteenth                          | June 19th\*
```

Comment:
@tatianahernandez for the table to be complete we need the end '|' symbols added.